packages/ui/src/components/Form/CustomAutoForm.tsx

```diff
@@ -72,6 +75,7 @@ export const CustomAutoForm = forwardRef<CustomAutoFormRef, CustomAutoFormProps>
           <AutoFields omitFields={props.omitFields} />
         )}
         <ErrorsField />
+        <button ref={props.submitRef} style={{ display: 'none' }} />
```

Comment:
not sure bout this, whether there are some issues it could cause or whether this is an allowed way to invoke the form submit.

Reply:
Not the clear answer but PatternFly guideline suggests to disable the button until the condition is satisfied - https://www.patternfly.org/components/forms/form/design-guidelines/#buttons

Reply:
no longer an issue - solved with @lordrip